src/styles/main.scss

```diff
@@ -708,3 +708,30 @@ Fixing styling for the code example tables in the "Migrating From..." pages
   top: calc(50% - 11px);
   left: 18px;
 }
+
+// CSS styling for the date modifiex X ago
+#postedon {
```

Comment:
dumb q. what is postedon meant to mean?

Reply:
posted-on

Reply:
oh can we change it to #timepostedon or #datepostedon? 

or just add a dash between posted and on?

Reply:
I can do that! no problem